<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BondQueryPopup</class>
 <widget class="QWidget" name="BondQueryPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="2">
    <widget class="QToolButton" name="single_double_btn">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Single or Double Bond</string>
     </property>
     <property name="text">
      <string>S/D</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="any_btn">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Any Bond</string>
     </property>
     <property name="text">
      <string>Any</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="aromatic_btn">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Aromatic Bond</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../sketcher.qrc">
       <normaloff>:/icons/bond_aromatic.svg</normaloff>
       <disabledoff>:/icons/bond_aromatic_dis.svg</disabledoff>:/icons/bond_aromatic.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="single_aromatic_btn">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Single or Aromatic Bond</string>
     </property>
     <property name="text">
      <string>S/A</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="double_aromatic_btn">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Double or Aromatic Bond</string>
     </property>
     <property name="text">
      <string>D/A</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../sketcher.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="group"/>
 </buttongroups>
</ui>
